gtda/diagrams/representations.py

```diff
@@ -700,6 +700,11 @@ def plot(self, Xt, sample=0, homology_dimension_idx=0, colorscale="blues",
             :meth:`update_traces` and :meth:`update_layout` methods of
             :class:`plotly.graph_objects.Figure`.
 
+        sample_orig : int or None, optional, default: ``None``
+            Needed when this method is called as part of a call to
+            :meth:`transform_plot`, to keep track of the original sample
+            index when generating the figure title. Do not use.
```

Comment:
The "Do not use" is a bit confusing, but i do not have a good suggestion (see general review comment).

Reply:
Maybe it is too strong, and "Not intended for direct use" would be better?

Reply:
Yes!